config/routes.rb

```diff
@@ -84,7 +84,6 @@
 
   namespace :super_admins do
     root to: 'providers#index'
-    concerns :provider_management_routes
```

Comment:
**not a blocker** but i had added this "route concern" in only because we were sharing between case workers and super admins. now that super admins are not to have provider management i'd get rid of this concern and put the routes directly into the relevant resources namespace for case workers

Reply:
👍